README.rst

```diff
@@ -112,7 +112,80 @@ Please run the following:
    If you do not have all the versions of python installed, just run tox on
    the versions you have. pep8 and pylint must run clean also.
 
-If it all looks good, I'll merge it.
+If it all looks good, I'll merge it.  Please refer to CONVENTIONS section for more details
+
+CONVENTIONS
+
+For neatness and readability we will enforce the following conventions going forward.
+
+1. Single quotes ' unless double quotes " necessary.
+
+2. use .format() when manipulating strings
+
+.. code-block::
+
+   my_string= '/{variable1}, thanks for contributing to {variable2}'.format(
+       variable1=’Hello’, variable2=’PyU4V’)
+
+3. We cannot use .format() in logging due to pylint error (W1202) so we follow the following format:
+
+.. code-block::
+
+   my_message = 'Hello, this is my log message.'
+   logger.debug('message: %(my_message)s', {my_message: my_message})
+
+4. Use :returns: in docstring.  Pep8 will guide you with all the other docstring conventions
+
+.. code-block::
+
+   """The is my summary of the method with full stop.
+
+   This is a brief description of what the method does.  Keep
+   it as simple as possible.
+
+   :param parameter1: brief description of parameter 1
+   :param parameter2: brief description of parameter 2
+   :returns: what gets returned from method, omit if none
+   :raises: Exceptions raised, omit if none
+    """
+
+5. Class name is mixed case with no underscores _
+
+.. code-block::
+
+   class ClassFunctions(object):
+       """Collection of functions ClassFunctions."""
+
+6. Public Methods are separated by underscores _.  Make the name as meaningful as possible
+
+.. code-block::
+
+    def public_function_does_exactly_what_it_says_it_does(self):
+        """Function does exactly what it says on the tin."""
+
+7. Public Methods are prefixed and separated by underscores _.  Make the name as meaningful as possible
```

Comment:
Should this be "Private Methods are prefixed..."

Reply:
Sure

Reply:
Fixed in commit 5f841c1b3ba0fd3c6f753d67d283ee3dba96809b. Resolving.